spec/workers/update_all_sources_worker_spec.rb

```diff
@@ -20,12 +20,12 @@
         ]
     }
 
-    it "should add news to the database" do
-        expect(News.count).to eq(0)
-        
-        UpdateAllSourcesWorker.perform_async
-        Sidekiq::Worker.drain_all # Isso força todos os workers a terminarem.
-
-        expect(News.count).to be > 0
+    it "should call NewsFeedUpdateWorker" do
```

Comment:
Tem como explicar levemente o que está rolando aqui? Basicamente, é o negócio que força atualizar as notícias no Heroku?

Reply:
Basicamente, eu testo se o `UpdateAllSourcesWorker` coloca jobs do tipo `NewsFeedUpdateWorker` na fila do Sidekiq (o nome do teste tá meio estranho, vou mudar isso depois).

É como se eu tivesse dado um mock no `NewsFeedUpdateWorker` pra testar se ele foi chamado, mas a maneira que o Sidekiq é usado dificulta usar mocks "normais".